### PR TITLE
Use `cli` for trusted relate command

### PR DESCRIPTION
**Pull request by vdemeester**
This also removed some skipped test (that are skipped for a long while).

🦁

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: integration-cli/docker_cli_create_test.go [hunk 1]
```diff
@@ -322,36 +320,10 @@ func (s *DockerTrustSuite) TestTrustedIsolatedCreate(c *check.C) {
...
-}
-
-func (s *DockerTrustSuite) TestCreateWhenCertExpired(c *check.C) {
-	c.Skip("Currently changes system time, causing instability")
-	repoName := s.setupTrustedImage(c, "trusted-create-expired")
-
-	// Certificates have 10 years of expiration
-	elevenYearsFromNow := time.Now().Add(time.Hour * 24 * 365 * 11)
-
-	testutil.RunAtDifferentDate(elevenYearsFromNow, func() {
```
Comment (dnephin) [inline integration-cli/docker_cli_create_test.go]: I think you've removed all calls of this functions. We should remove the function as well.

Comment (vdemeester) [inline integration-cli/docker_cli_create_test.go]: oh right, didn't think of that. I'll look 😉

**Comment by vdemeester** [comment]
Failure fixed by #32663

**Comment by tonistiigi** [comment]
@vdemeester Ci failures

**Pull request closed without merging**